ocrd/ocrd/workspace_bagger.py

```diff
@@ -1,12 +1,15 @@
 from datetime import datetime
 from os import makedirs, chdir, walk
 from os.path import join, isdir, basename, exists, relpath
+from pathlib import Path
 from shutil import make_archive, rmtree, copyfile, move
-from tempfile import mkdtemp
+from tempfile import mkdtemp, TemporaryDirectory
 import re
 import tempfile
 import sys
-from bagit import Bag, make_manifests, _load_tag_file  # pylint: disable=no-name-in-module
+from bagit import Bag, make_manifests, _load_tag_file, _make_tag_file, _make_tagmanifest_file  # pylint: disable=no-name-in-module
+from distutils.dir_util import copy_tree
+from contextlib import nullcontext
```

Comment:
Even if we do not _support_ 3.6 anymore, we should not try to actively _break_ things unnecessarily, forcing users to update at that particular time.
```suggestion
try:
    from contextlib import nullcontext
except ImportError:
    from contextlib import contextmanager
    @contextmanager
    def nullcontext(enter_result=None):
        yield enter_result
```

Reply:
I agree to try to not break compatibility if it can be avoided. But tbh I find that import error handling "suboptimal" and hard to read. What do you think about just removing the nullcontext? Then there will be cases when a temporary directory is created but not used but it gets deleted afterwards anyway:
```
with TemporaryDirectory() as tempdir:
```
(I tried, but I think it is not possible to response with a change suggestion on another location here)

Reply:
oh, absolutely! Did not see it myself. Should definitely work, much better that way.

Reply:
I removed the nullcontext as discussed with 6d13bbe4d. (btw: I don't see a possibility to commit parts of your suggestion or to somehow use github-tools to do this commit with better reference to this discussion. Please correct me if I am wrong and there is a better way to make this code-change)